### PR TITLE
doc: Add known issue for DRGN-23231 (deadlock on link loss)

### PR DESCRIPTION
**Pull request by rugeGerritsen**
Adds an issue which appeared as a result of the fix in DRGN-21085. The issue is fixed for NCS 2.8.0

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst
Comment (rugeGerritsen) [inline doc/nrf/releases_and_maturity/known_issues.rst:1]: Should we also add an entry to the release notes stating that this issue has been fixed?

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -144,6 +144,15 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-4-4 v2-4-3
+
+DRGN-23302: The Bluetooth subsystem may in some cases deadlock when a Bluetooth link is lost while there is data transfer ongoing.
+  When this happens, the disconnected event is never delivered to the application.
+  The issue occurs when :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` is enabled which is enabled by default on nRF5340.
+  The issue is a result of the fix applied in DRGN-21085.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/known_issues.rst:152]:
```suggestion
DRGN-23302: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link is lost during data transfer
  When this happens, the disconnected event is never delivered to the application.
  The issue occurs when the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is enabled.
  This option is enabled by default on the nRF5340 DK.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -144,6 +144,15 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
...
+
+  **Affected platforms:** nRF5340
+
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/known_issues.rst:155]: Any workaround?

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 2 above)
Comment (rugeGerritsen) [reply doc/nrf/releases_and_maturity/known_issues.rst:155]: Resetting the device is the only thing that resolves it, so there is no good workaround for it

**Review by peknis — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst
Comment (rugeGerritsen) [reply doc/nrf/releases_and_maturity/known_issues.rst:1]: Done

**Review by vidarbe — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -144,6 +144,16 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-4-4 v2-4-3
+
+DRGN-23231: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link is lost during data transfer.
+  When this happens, the disconnected event is never delivered to the application.
+  The issue occurs when the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is enabled.
+  This option is enabled by default on the nRF5340 DK.
+  The issue is a result of the fix applied in DRGN-21085.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/known_issues.rst:153]:
```suggestion
DRGN-23231: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link is lost during data transfer
  When this happens, the disconnected event is never delivered to the application.
  The issue occurs when the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is enabled.
  This option is enabled by default on the nRF5340 DK.
```
We do not use full stop on the first line (the short description of the issue.
And I think we do not want to advertise that an issue occurs due to a fix done earlier.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by alwa-nordic — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -142,6 +142,10 @@ Amazon Sidewalk
 Bluetooth® LE
 -------------
 
+* Fixed an issue where the Bluetooth subsystem deadlocked when a Bluetooth link was lost during data transfer.
+  In this scenario the disconnected event was never delivered to the application.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:146]:
```suggestion
  In this scenario, the disconnected event was never delivered to the application.
```

**Review by peknis — APPROVED** [review]
Approved with a minor nit that I overlooked earlier.

**Review by peknis — APPROVED** [review]

**Pull request merged**